packages/api/openapi/parameters/order-by-param.yml

```diff
@@ -2,7 +2,7 @@ x-template: &orderByParam
   name: orderBy
   in: query
   description: The order of the result set.
-  example: 'createdAt~desc'
+  example: ['createdAt~desc']
```

Comment:
Jag är inte säker på att det här stämmer. Man skickar väl in en array i queryn genom att ange queryn flera gånger snarare?

```
orderBy=createdAt~desc&orderBy=updatedAt~asc
```

Reply:
Den är typad som array två rader under, därför jag korrigerade exempelt till en array.

I swagger-editorn anger du dem som en lista, men de blir konverterade till chainade queries, se bild:
![image](https://user-images.githubusercontent.com/33149910/185401085-97e2c452-5d1e-4d28-afb7-2fac06044b51.png)